src/Components/Facility/PatientNotesSlideover.tsx

```diff
@@ -37,6 +36,11 @@ export default function PatientNotesSlideover(props: PatientNotesProps) {
   const { facilityId, patientId, consultationId, setShowPatientNotesPopup } =
     props;
 
+  const localStorageKey = `patientNotesNoteField_${patientId}`;
```

Comment:
Its better to keep it consultation-specific rather than patient-specific since a patient can have multiple consultations.